src/couch/src/couch_db.erl

```diff
@@ -1138,8 +1068,8 @@ update_docs(Db, Docs0, Options, interactive_edit) ->
         Options2 = if AllOrNothing -> [merge_conflicts];
                 true -> [] end ++ Options,
         DocBuckets3 = [[
-                doc_flush_atts(set_new_att_revpos(
-                        check_dup_atts(Doc)), Db#db.fd)
+                doc_flush_atts(Db, set_new_att_revpos(
```

Comment:
There are an old bit in [`set_new_att_revpos/1`](https://github.com/apache/couchdb/blob/7162a24ba6b6341a9c015a5fc6b6f4d4306ba432/src/couch/src/couch_db.erl#L1210) function, when it expects `couch_att:fetch(data, Att)` to return `{Fd, Sp}` instead of new response `{stream, Engine}`. Vars there set with `_` to be ignored, so it works, but it'd be nice to update for sake of the future readers.